src/libs/actions/Transaction.ts

```diff
@@ -59,7 +59,7 @@ function addStop(transactionID: string) {
 function saveWaypoint(transactionID: string, index: string, waypoint: RecentWaypoint | null, isEditingWaypoint = false) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`, {
         pendingFields: {
-            comment: isEditingWaypoint ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+            waypoints: isEditingWaypoint ? CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE : CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
```

Comment:
In here we should update pendingFields.waypoints like we did when changing waypoint in EditRequestPage

https://github.com/Expensify/App/blob/0888bdaf91c5414f586bda252ba2751da487de90/src/libs/actions/IOU.js#L689

I log pendingFields here anh Let's see this video

https://github.com/Expensify/App/assets/141406735/2e09fb13-20d3-49ad-8aae-056ac4e81be1